native/packages/react-native-bpk-component-button-link/readme.md

```diff
@@ -0,0 +1,65 @@
+# react-native-bpk-component-button-link
+
+> Backpack React Native button link component.
+
+## Installation
+
+1. Install the NPM package:
+```sh
+npm install react-native-bpk-component-button-link --save-dev
+```
+
+
+3. `react-native-bpk-component-button-link` depends on [react-native-bpk-component-icon](https://github.com/Skyscanner/backpack/tree/master/native/packages/react-native-bpk-component-icon) make sure to follow the installation guide for the icon component for the different platforms
```

Comment:
Might be better to make this link point to the one on npm? Feel free to disagree though.

Reply:
No make sense, it's copied from buttons tough so I might need to update that one as well, wdyt?

Reply:
Yeah good shout